frontend/qawwali-tradition/style.css

```
@@ -0,0 +1,263 @@
+    color: var(--primary-color);
+    font-size: 1.8rem;
+    margin-bottom: 1rem;
+}
+
+.instrument-desc p {
+    color: var(--text-color);
+    font-size: 1.1rem;
+    line-height: 1.7;
+}
+
+@keyframes fadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)